deploy/docker-swarm/pkg/Makefile

```diff
@@ -0,0 +1,32 @@
+# FIXME: change this
+PLUGIN_NAME = hetznercloud/csi-driver-docker
```

Comment:
I would prefer to use the same docker image as for the kubernetes(/general) csi support: `hetznercloud/hcloud-csi-driver`. To differentiate between the normal and Docker Swarm images, we can use tags:

- `hetznercloud/hcloud-csi-driver:latest` -> normal
- `hetznercloud/hcloud-csi-driver:latest-swarm` -> Docker Swarm

Reply:
Due to the nature of how docker plugins are packaged, I don't think that this is possible or even something we would want. See https://github.com/hetznercloud/csi-driver/pull/376#discussion_r1104137481